patches/graphql-relay+0.5.3.patch

```diff
@@ -0,0 +1,31 @@
+patch-package
+--- a/node_modules/graphql-relay/lib/connection/arrayconnection.js
++++ b/node_modules/graphql-relay/lib/connection/arrayconnection.js
+@@ -61,7 +61,8 @@ function connectionFromArraySlice(arraySlice, args, meta) {
+       first = args.first,
+       last = args.last;
+   var sliceStart = meta.sliceStart,
+-      arrayLength = meta.arrayLength;
++      arrayLength = meta.arrayLength,
++      resolveNode= meta.resolveNode;
```

Comment:
Missing spacey

Reply:
Doh, I actually turned off `formatOnSave` since it was removing semicolons and doing quite a number on this poor little `node_module/` file.

Reply:
Oh yeah, I can imagine. I actually have an [extension](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.save-without-format) that allows you to bypass selectively installed for that. In the meantime, I can be your local linter cop 👮